app/src/main/res/values/strings.xml

```diff
@@ -1,27 +1,27 @@
 <resources>
     <string name="app_name">Storyous Assistant</string>
-    <string name="no_permissions_camera">Aplikace potřebuje použít fotoaparát k jednorázovému načtení QR kódu z pokladny.</string>
-    <string name="give_camera_permission">Povolit přístup ke fotoaparátu</string>
-    <string name="remove_configuration">Odpárovat pokladnu</string>
-    <string name="synchronize">Propisovat telefonní čísla do poklady</string>
-    <string name="notification_info">Vaše příchozí hovory se nyní synchronizují do pokladního systému Storyous.</string>
-    <string name="config_error">Nastala chyba při konfiguraci. Zkuste to prosím později.</string>
+    <string name="no_permissions_camera">Storyous Assistant requires the use of your camera in order to scan a QR code shown in your POS.</string>
+    <string name="give_camera_permission">Allow access to camera</string>
+    <string name="remove_configuration">Unpair POS</string>
+    <string name="synchronize">Sync call informations to POS</string>
+    <string name="notification_info">Your incoming phone calls are now being synced with POS.</string>
+    <string name="config_error">An error occurred, please try again later.</string>
 
     <string-array name="config_instructions">
-        <item>Spárování aplikace s pokladním systémem Storyous:</item>
-        <item>V pokladním systému Storyous otevřete menu pomoct tlačítka v levém horním rohu.</item>
-        <item>Otevřete sekci „Zařízení”.</item>
-        <item>Vyberte Storyous Assistant.</item>
-        <item>Zobrazí se vám QR kód, který tímto zařízením načtěte.</item>
+        <item>How to pair Storyous Assistant with POS</item>
+        <item>Go to Storyous POS and open the left side menu clicking on the top left menu button</item>
+        <item>Click on "Devices"</item>
+        <item>Click on "Storyous Assistant"</item>
+        <item>Use this app to scan the QR code that will be displayed in POS</item>
     </string-array>
 
     <string-array name="create_order_instructions">
-        <item>Synchronizace je zapnutá. Telefonní číslo, které vám bude volat na tento telefon lze načíst přímo z pokladny.</item>
-        <item>Podržením na „Rychlá objednávka” otevřete okno pro zadání objednávky.</item>
-        <item>V sekci “telefonní číslo” vyberte číslo, se kterým právě telefonujete.</item>
-        <item>Pokud jste už v minulosti k tomuto číslu předvyplnili ostatní informace, pokladní systém je nyní doplní automaticky, v případě si informace uloží pro příští objednávku.</item>
-        <item>Aplikace běží na pozadí, nemusíte ji tedy mít pro správnou funkci aktivní.</item>
+        <item>Synchronization is enabled. When a customer calls you on this phone, his phone number will be shared with POS.</item>
+        <item>Touch and hold on "Quick order" button. Customer delivery info will be shown. </item>
```

Comment:
Nevim, jestli by nebylo lepší "Touch and hold Quick order button" bez toho "on"

Reply:
👍